app/src/main/java/io/homeassistant/companion/android/bluetooth/ble/IBeaconMonitor.kt

```diff
@@ -21,17 +21,32 @@ data class IBeacon(
 class IBeaconMonitor {
     lateinit var sensorManager: BluetoothSensorManager
     var beacons: List<IBeacon> = listOf()
+    var lastSeenBeacons: Collection<Beacon> = listOf()
+    private var uuidFilter: List<String> = listOf()
+    private var uuidFilterExclude: Boolean = false
```

Comment:
This type can be inferred

```suggestion
    private var uuidFilterExclude = false
```